README.md

```diff
@@ -8,32 +8,27 @@ I want to install Mattermost on my Raspberry Pi for use on my network.
 
 The Raspberry Pi architecture is not officially supported by Mattermost, but there is an excellent resource available with steps and helpful links [Mattermost on Raspberry Pi](https://kartoffelsalat.ddns.net/post/mattermost-raspi/), which further linked me to a really great repo with the latest install files provided by [SmartHoneyBee](https://github.com/SmartHoneybee/ubiquitous-memory/releases/). 
 
-The builds are updated regularly, so the most recent version of Mattermost (5.21) is available. There are installers for quite a few flavors of Linux. The version I used was **Linux-arm-tar.gz** as I'm running Raspbian 9 (stretch) on my Raspberry Pi 3 Model B which has an ARM Cortex-A53 processor. You can check this by running *uname --kernel-name --kernel-release --machine*. If your Pi is out the box, the results should be something like **Linux 4.14.34-v7+ arm71**. If you've installed a different flavor of Linux, it'll be different so choose the appropriate install file.  
+The builds are updated regularly, so the most recent version of Mattermost (5.21) is available. There are installers for quite a few flavors of Linux. The version I used was **Linux-arm-tar.gz** as I'm running Raspbian 10 (buster) on my Raspberry Pi 4 Model B which has an ARM. You can check this by running *uname --kernel-name --kernel-release --machine*. If your Pi is out the box, the results should be something like **Linux 4.19.97-v7l+ armv7l**. If you've installed a different flavor of Linux, it'll be different so choose the appropriate install file.  
 
-Setting up MySQL
-----------------
-Before you install Mattermost, you need to set up MySQL. The full instructions are on the [Installing Mattermost on Debian Stretch](https://docs.mattermost.com/install/install-debian.html/) page and cover PostgreSQL and MySQL. I use MySQL, so these are the steps I followed: 
+Setting up MariaDB
+------------------
+Before you install Mattermost, you need to set up MariaDB. The full instructions are on the [Installing Mattermost on Debian Stretch](https://docs.mattermost.com/install/install-debian.html/) page and cover PostgreSQL and MySQL. I use MariaDB (a drop-in replacement of MySQL), so these are the steps I followed: 
 
 1. Log into the server that will host the database, and open a terminal window.
 
-2. Download the MySQL repository package 
+2.  your local package list
```

Comment:
```suggestion
2.  Download your local package list
```

Reply:
Well, how about "Update your local package list", instead of "Download your local package list"?